prow/plugins/require-matching-label/require-matching-label.go

```diff
@@ -95,14 +99,24 @@ func helpProvider(config *plugins.Configuration, _ []config.OrgRepo) (*pluginhel
 	if err != nil {
 		logrus.WithError(err).Warnf("cannot generate comments for %s plugin", pluginName)
 	}
-	return &pluginhelp.PluginHelp{
-			Description: `The require-matching-label plugin is a configurable plugin that applies a label to issues and/or PRs that do not have any labels matching a regular expression. An example of this is applying a 'needs-sig' label to all issues that do not have a 'sig/*' label. This plugin can have multiple configurations to provide this kind of behavior for multiple different label sets. The configuration allows issue type, PR branch, and an optional explanation comment to be specified.`,
-			Config: map[string]string{
-				"": fmt.Sprintf("The plugin has the following configurations:\n<ul><li>%s</li></ul>", strings.Join(descs, "</li><li>")),
-			},
-			Snippet: yamlSnippet,
+	pluginHelp := &pluginhelp.PluginHelp{
+		Description: `The require-matching-label plugin is a configurable plugin that applies a label to issues and/or PRs that do not have any labels matching a regular expression. An example of this is applying a 'needs-sig' label to all issues that do not have a 'sig/*' label. This plugin can have multiple configurations to provide this kind of behavior for multiple different label sets. The configuration allows issue type, PR branch, and an optional explanation comment to be specified.`,
+		Config: map[string]string{
+			"": fmt.Sprintf("The plugin has the following configurations:\n<ul><li>%s</li></ul>", strings.Join(descs, "</li><li>")),
 		},
-		nil
+		Snippet: yamlSnippet,
+	}
+
+	pluginHelp.AddCommand(pluginhelp.Command{
+		Usage:       "/check-required-labels",
+		Description: "Forces the require-matching-label plugin to reprocess an issue or PR that do not have any labels matching.An example of this is applying a 'needs-sig' label to all issues that do not have a 'sig/*' label",
```

Comment:
Others please feel free to chime in. I'm still not sure about this description:
```suggestion
              Description: "Forces the require-matching-label plugin to re-check an issue or PR to add any labels it's missing. An example of this is applying a 'needs-sig' label to all issues that do not have a 'sig/*' label",
```